examples/nextjs/starter/package.json

```diff
@@ -19,36 +19,36 @@
 		"test:watch": "jest --watch"
 	},
 	"dependencies": {
-		"@apollo/client": "^3.12.8",
+		"@apollo/client": "^3.13.7",
 		"@snapwp/blocks": "file:../../../packages/blocks",
 		"@snapwp/core": "file:../../../packages/core",
 		"@snapwp/next": "file:../../../packages/next",
 		"@snapwp/query": "file:../../../packages/query",
 		"graphql": "^16.10.0",
-		"next": "^15.2.3",
-		"react": "^19.0.0",
-		"react-dom": "^19.0.0"
+		"next": "^15.3.0",
+		"react": "^19.1.0",
+		"react-dom": "^19.1.0"
 	},
 	"devDependencies": {
-		"@graphql-codegen/cli": "^5.0.4",
+		"@graphql-codegen/cli": "^5.0.5",
 		"@parcel/watcher": "^2.5.1",
 		"@snapwp/codegen-config": "file:../../../packages/codegen-config",
 		"@snapwp/eslint-config": "file:../../../packages/eslint-config",
 		"@snapwp/prettier-config": "file:../../../packages/prettier-config",
 		"@testing-library/jest-dom": "^6.6.3",
-		"@testing-library/react": "^16.2.0",
+		"@testing-library/react": "^16.3.0",
 		"@types/jest": "^29.5.14",
 		"@types/node": "^20",
-		"@types/react": "^19.0.8",
-		"@types/react-dom": "^19.0.3",
+		"@types/react": "^19.1.2",
+		"@types/react-dom": "^19.1.2",
 		"concurrently": "^9.1.2",
-		"dotenv": "^16.4.7",
+		"dotenv": "^16.5.0",
 		"eslint": "^8",
-		"eslint-config-next": "14.2.5",
+		"eslint-config-next": "^15.3.0",
```

Comment:
Not sure why this was set to v14.x to begin with. Assuming it was a bug.

Reply:
Looks like it.